src/tests/test_utils.py

```diff
@@ -1140,3 +1140,34 @@ def test_get_genome_uuid_by_tag(self, genome_db_conn, genome_tag, expected_outpu
 				genome_tag=genome_tag,
 			))
 		assert json.loads(output) == expected_output
+
+
+	def test_ftp_links(self, genome_db_conn):
```

Comment:
It will be nice to add the case where `genome_uuid` isn't valid or doesn't exist (we will be expecting an empty Links list)

Reply:
Thanks @bilalebi. Added now.